packages/procaptcha-react/src/components/Procaptcha.tsx

```diff
@@ -205,52 +217,44 @@ export const Procaptcha = (props: ProcaptchaProps) => {
-                                                    left: 'auto',
-                                                    opacity: '1',
-
-                                                    '&::before': {
-                                                        content: '""',
-                                                        position: 'absolute',
-                                                        height: '100%',
-                                                        width: '100%',
-                                                    },
-                                                },
+                                            //inputProps={{ 'aria-label': 'controlled' }}
+                                            style={{
+                                                width: '1.4em',
+                                                height: '1.4em',
+                                                top: 'auto',
+                                                left: 'auto',
+                                                opacity: '1',
                                             }}
                                         />
-                                    </Box>
-                                    <Box
-                                        sx={{
+                                    </div>
+                                    <div
+                                        style={{
                                             display: state.loading ? 'block' : 'none',
                                         }}
                                     >
-                                        <Box pt={'5px'}>
-                                            <CircularProgress size={'24px'} disableShrink />
-                                        </Box>
-                                    </Box>
-                                </Box>
-                                <Box p={1}>
-                                    <Typography color={theme.palette.primary.contrastText}>I am a human</Typography>
-                                </Box>
-                            </Box>
-                        </Box>
-                        <Box>
-                            <Link href="https://prosopo.io" target="_blank">
-                                <Box>
+                                        <div style={{ paddingTop: '5px' }}>
+                                            <LoadingSpinner themeColor={themeColor} />
+                                        </div>
+                                    </div>
+                                </div>
+                                <div style={{ padding: 1 }}>
+                                    <span style={{ color: theme.palette.primary.contrastText }}>I am a human</span>
+                                </div>
+                            </div>
+                        </div>
+                        <div>
+                            <a href="https://prosopo.io" target="_blank">
```

Comment:
It could be smart to change this to a FAQ for Procaptcha, then set rel="help"

Reply:
Very good point.